src/arrayMethodJoin.js

```diff
@@ -4,8 +4,20 @@
  * Implement method join
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
-    // write code here
+  [].__proto__.join2 = function(separator = ',') {
+    let sum = '';
```

Comment:
for better code practice use semantic names please, it is not a `sum`, it is string or result, name it accordingly

Reply:
corrected